Userland/Libraries/LibGfx/ClassicStylePainter.cpp

```diff
@@ -303,7 +304,7 @@ void ClassicStylePainter::paint_window_frame(Painter& painter, IntRect const& re
         // FIXME: This will draw "useless" pixels that'll get drawn over by the window contents.
         // preferrably we should just remove the corner pixels from the completely drawn window
         // but I don't know how to do that yet. :^)
-        painter.fill_rect_with_rounded_corners(rect, base_color, border_radius);
+        AntiAliasingPainter { painter }.fill_rect_with_rounded_corners(rect, base_color, border_radius);
```

Comment:
We don't usually initialize and call a method on it immediately like this (that I can recall, at least). Perhaps just split this into 2 lines?